### PR TITLE
Remove old download links

### PR DESCRIPTION
**Pull request by jkonrath-postman**

### CODE REVIEW

File: src/components/LeftNav/LeftNavItems.jsx [hunk 1]
```diff
@@ -1354,10 +1354,6 @@ export const leftNavItems = [
           }
         ],
       },
-      {
-        name: 'Updating your team to the current version of Postman',
-        url: '/docs/administration/updating/',
-      },
```
Comment (MatthewPinsker-postman) [inline src/components/LeftNav/LeftNavItems.jsx:1360]: I'm also seeing about 7 redirects that point to this page in `redirects.json`. Should we delete those redirects as well to declutter the file?

**Review by nerminamiller-postman — APPROVED** [review]
LGTM!

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM! I added a question about cleaning up the `redirects.json` file.

File: src/components/LeftNav/LeftNavItems.jsx (hunk 1 above)
Comment (jkonrath-postman) [reply src/components/LeftNav/LeftNavItems.jsx:1360]: I changed redirects to point to the main installing and updating page

File: redirects.json [hunk 2]
```diff
@@ -21,11 +21,11 @@
   },
   {
     "from": "/docs/administration/migrating_to_v7/",
```
Comment (MatthewPinsker-postman) [inline redirects.json:23]: Redirect updates work as expected!

**Question**: If we're updating these existing redirects, should we also add a new redirect to point from `/docs/administration/updating/` to `/docs/getting-started/installation/installation-and-updates/`?

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM! I tested redirects and they work as expected. Also I added a follow-up question about redirects.

**Pull request merged**